package.xml

```diff
@@ -18,6 +18,7 @@
 
   <build_depend>boost</build_depend>
   <build_depend>pluginlib</build_depend>
+  <build_depend>qtbase5-dev</build_depend>
```

Comment:
I don't think there is a need to branch out. This package has been released before and is working fine with other Ubuntu distros. I would expect that the system packages have slightly changed and the rosdep keys need to be reviewed and updated for Zesty.